source/Missions/Services/Agents/Packets/MovementHandler.cs

```diff
@@ -35,6 +36,12 @@ public MovementPacket(Guid agentGuid, Agent agent)
             Agent = new AgentData(agent);
         }
 
+        public MovementPacket(Guid agentGuid, AgentData agentData)
```

Comment:
Replace movement packet with movement delta class. Also rename "AgentMovementDelta" to "AgentMovement" as it really doesn't contain deltas rather only actual values of data that has changed